codes.json

```diff
@@ -716,6 +716,22 @@
       "en": "An environment variable is missing in the configuration."
     }
   },
+  "Invalid_MapMonitoring_Type": {
+    "status": 400,
+    "code": 4000084,
+    "error": "invalid_mapmonitoring_type",
+    "description": {
+      "en": "MapMonitoring type must be either pricing, multiplierbycollection, or multiplierbyrecord (case insensitive)."
+    }
+  },
+  "Invalid_MapMonitoring_Status": {
```

Comment:
same case here , this value need to match the value in the 'constanst' object @husseiny-m